apps/onboarding-functions/src/main/java/it/pagopa/selfcare/onboarding/functions/NotificationFunctions.java

```diff
@@ -98,4 +102,57 @@ public HttpResponseMessage resendNotification(
         notificationEventService.send(context, onboarding.get(), queueEvent);
         return request.createResponseBuilder(HttpStatus.OK).build();
     }
+
+
+    @FunctionName("FindNotifications")
```

Comment:
you could add some comments about needed this function, why do we need to find a "notification" ? we could use the find onboarding endpoint for example